docs/recipes/puppeteer.md

```diff
@@ -0,0 +1,43 @@
+# Testing web apps using Puppeteer
+
+## Dependencies
+
+- [Puppeteer](https://github.com/GoogleChrome/puppeteer)
+	- `npm i --save-dev puppeteer`
+
+## Setup
+
+The first step is setting up a helper to configure the environment:
+
+`./test/helpers/setup.js`
+
+```js
+const { test } = require('ava');
+const puppeteer = require('puppeteer');
+const url = "https://google.com"; // App URL, for example, google.com
+
+global.test = test;
+
+test.before(async () => {
+  global.browser = await puppeteer.launch();
+  global.page = await browser.newPage();
```

Comment:
We're not very keen on globals with AVA. Instead, export a `setup` function that takes a `test` object. Then use that to hook up `before` and `after`.

This function also gets a `t` object. I would assign `browser` and `page` to `t.context.browser`, etc. Then in the tests you can access them via `t.context` and you can avoid the globals.